Project-Timer-frontend/src/app/login-page/login-page.component.html

```diff
@@ -0,0 +1,30 @@
+<form class="example-form" #loginForm="ngForm">
```

Comment:
poczytaj o metodyce BEM
https://www.nafrontendzie.pl/metodyki-css-2-bem
 polecam, jeśli Ci przypadnie do gustu to nazywaj klasy w html według tej metodyki

Reply:
Ta metodyka wydaje mi się nie czytelna, podążając za przykładem stamtąd dlaczego robić:
`login__input--password {style}`
jeśli mogę zrobić:
`input.login input.password {style}`

Reply:
Będę używał OOCSS